docs/build/guides/conventions/upgrading-contracts.mdx

```diff
@@ -74,7 +90,13 @@ pub fn upgrade(e: Env, new_wasm_hash: BytesN<32>) {
 }
 ```
 
-The `update_current_contract_wasm` host function will also emit a `SYSTEM` contract [event] that contains the old and new wasm reference, allowing downstream users to be notified when a contract they use is updated. The event structure will have `topics = ["executable_update", old_executable: ContractExecutable, old_executable: ContractExecutable]` and `data = []`.
+- `e: Env`: The environment object representing the current state of the blockchain.
+- `new_wasm_hash: BytesN<32>`: The hash of the new Wasm code for the contract.
+- The function first retrieves the admin's address from the contract's storage.
+- It then requires the admin's authorization (`admin.require_auth()`) to proceed.
+- Finally, it updates the contract with the new Wasm code (`e.deployer().update_current_contract_wasm(new_wasm_hash)`).
```

Comment:
Let's clarify that it's the **contract instance** that is updated to "point to" the installed bytecode (identified by the hash). I think I worded that badly, but I hope it makes sense lol